tests/itests/src/test/java/org/codice/ddf/admin/query/request/LdapRequestHelper.java

```diff
@@ -23,6 +23,7 @@
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
 import org.boon.Boon;
+import org.codice.ddf.admin.comp.graphql.GraphQlHelper;
```

Comment:
Is this actually needed?

Reply:
This class needs to be exported.

Reply:
ah, didn't see that you moved the class :+1: